.github/workflows/nightly.yml

```diff
@@ -79,7 +76,7 @@ jobs:
       # Always run this step (otherwise it would be skipped if any of the previous steps fail) but only if the
       # `install` or `tests` steps failed, and the `SLACK_WEBHOOK` is available. The latter is not the case for
       # pull requests that come from forks. This is a limitation of secrets on GHA
-      if: always() && (steps.install.outcome == 'Failure' || steps.tests.outcome == 'Failure') && env.SLACK_WEBHOOK != null
+      if: always() && (steps.install.outcome == 'failure' || steps.tests.outcome == 'failure') && env.SLACK_WEBHOOK != null
```

Comment:
Is this essentially the fix?

Reply:
Yep!

Reply:
Also the steps in rabbitmq-tests needed ids.